src/test/java/be/seeseemelk/mockbukkit/entity/CamelMockTest.java

```diff
@@ -0,0 +1,71 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkitExtension;
+import be.seeseemelk.mockbukkit.MockBukkitInject;
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.entity.Camel;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Horse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockBukkitExtension.class)
+class CamelMockTest
+{
+
+	private Camel camel;
+	@MockBukkitInject
+	private ServerMock server;
+
+	@BeforeEach
+	void setUp()
+	{
+		camel = new CamelMock(server, UUID.randomUUID());
+	}
+
+	@Test
+	void testGetVariant()
+	{
+		assertEquals(Horse.Variant.CAMEL, camel.getVariant());
+	}
+
+	@Test
+	void testGetType()
+	{
+		assertEquals(EntityType.CAMEL, camel.getType());
+	}
+
+	@Test
+	void testIsDashingDefault()
+	{
+		assertFalse(camel.isDashing());
+	}
+
+	@Test
+	void testIsDashing()
+	{
+		camel.setDashing(true);
+		assertTrue(camel.isDashing());
+	}
```

Comment:
We could just theoretically include a test to see if setting dashing back to false also reverts back the isDashing condition, might be a little overkill and unnecessary, it does cover the other condition.

Reply:
We don't do that anywhere else, so it'd have to be updated everywhere for consistency, and that's out of the scope of this PR. The odds of that being an issue are extremely negligible, so we'll likely never do it.